drivers/hmis/app/graphql/mutations/create_duplicate_form_definition.rb

```diff
@@ -0,0 +1,67 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class CreateDuplicateFormDefinition < CleanBaseMutation
```

Comment:
I think just "DuplicateFormDefinition" would be fine and more succinct, but don't have a strong feeling either way